### DISCUSSION TITLE
About custom submit parameters and return parameters

### DISCUSSION THREAD

**Discussion by tianxiaode** [vitmalina/w2ui#2321] [Ideas]
Is it possible to set default parameters such as limit, skip, total, and records that allow users to modify and then submit and get data based on the user's settings?

**Comment by vitmalina** [comment]
It seems to me it is just properties on the grid that you can change, https://w2ui.com/web/docs/2.0/w2grid.limit, https://w2ui.com/web/docs/2.0/w2grid.skip - if you set it, doesn't it work?

**Reply by tianxiaode** [reply]
I mean that keywords skip, limit can be customized by the user, because I use a system that is not submitted using these as keywords, but this problem has been solved using the beofer submit event.
